### DISCUSSION TITLE
404: NOT_FOUND Code: DEPLOYMENT_NOT_FOUND

### DISCUSSION THREAD

**Discussion by yorhan0** [vercel/community#5827] [Help]
### Summary

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND

I uploaded my project and I get this error and please help me

### Example

404: NOT_FOUND Code: DEPLOYMENT_NOT_FOUND

### Steps to Reproduce

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND

**Comment by versecafe** [comment]
Make sure the URL you are checking is correct, and that your project successfully built and has a deployment preview, if a project is deleted or primary deployment URL is changed you'll end up with an error page like this

<img width="1361" alt="image" src="https://github.com/vercel/community/assets/147033096/043a7f34-c209-4e0b-8c93-79f188e32f6d">
